aten/src/THC/THCTensor.cpp

```diff
@@ -81,6 +89,81 @@ void THCTensor_resizeAs(THCState *state, THCTensor *self, THCTensor *src) {
     THCTensor_resizeNdLegacy(state, self, src->_dim(), src->size, NULL);
 }
 
+void THCTensor_resizeNd(THCState *state, THCTensor *self, int nDimension, int64_t *size, int64_t *stride)
```

Comment:
I cannot wait for THTensor and THCTensor to be the same thing and then we don't need to copy paste the code.